test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -447,6 +449,7 @@ TEST_F(SimpleHttpCacheTest, UpdateHeadersSkipSpecificHeaders) {
       {"content-length", "800"},
       {"age", "20"},
       {"etag", "0000-0000"},
+      {"etag", "1111-1111"},
```

Comment:
Sorry I'm actually confused now by your test structure. Could you rename response_headers_3 to expected_response_headers, and response_headers_2 to origin_response_headers, or similar? It's possible that response_headers_2 is actually cached_response_headers; not clear from looking at this code snippet.

Just numbering them doesn't show their

Reply:
For sure! Sorry about the confusion

Reply:
Fixed 😀